package.json

```diff
@@ -37,6 +38,7 @@
     "@testing-library/react": "^13.0.0",
     "@testing-library/react-17": "npm:@testing-library/react@12.1.4",
     "@testing-library/react-hooks": "^7.0.2",
+    "@testing-library/svelte": "^3.2.2",
```

Comment:
this should probably, if possible, just be a dependency of the svelte-query package